toolchain/driver/llvm_subcommand.h

```diff
@@ -0,0 +1,57 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef CARBON_TOOLCHAIN_DRIVER_LLVM_SUBCOMMAND_H_
+#define CARBON_TOOLCHAIN_DRIVER_LLVM_SUBCOMMAND_H_
+
+#include "common/command_line.h"
+#include "llvm/ADT/SmallVector.h"
+#include "llvm/ADT/StringRef.h"
+#include "toolchain/base/llvm_tools.h"
+#include "toolchain/driver/driver_env.h"
+#include "toolchain/driver/driver_subcommand.h"
+
+namespace Carbon {
+
+// Options for the LLVM subcommand.
+//
+// See the implementation of `Build` for documentation on members.
+struct LLVMOptions {
+  // Build the LLVM subcommand options using `b`. Whatever eventual nested
+  // subcommand action is selected, it needs to additionally run `select_fn`.
+  auto Build(CommandLine::CommandBuilder& b, DriverSubcommand* subcommand,
+             DriverSubcommand** selected_subcommand) -> void;
+
+  LLVMTool subcommand_tool;
+  llvm::SmallVector<llvm::StringRef> args;
+};
+
+// Implement the LLVM subcommand of the driver.
+//
+// This provides access to the full collection of internal LLVM command line
+// tools.
```

Comment:
"Internal" threw me a bit. How about instead "... full collection of LLVM command line tools built into the driver."?

(Sorry, GH isn't letting me suggest an edit for some reason)

Reply:
Simplified.